-------------------------------------------------------------------------------
Test set: com.sgg.covid19.dao.ProvinceMoveDaoTest
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.09 s <<< FAILURE! - in com.sgg.covid19.dao.ProvinceMoveDaoTest
testQueryProvinceMoveList(com.sgg.covid19.dao.ProvinceMoveDaoTest)  Time elapsed: 0.064 s  <<< ERROR!
org.springframework.jdbc.BadSqlGrammarException: 

### Error querying database.  Cause: java.sql.SQLSyntaxErrorException: Table 'covid19.tb_province_move' doesn't exist
### The error may exist in file [D:\covid19\target\classes\mapper\ProvinceMove.xml]
### The error may involve com.sgg.covid19.dao.ProvinceMoveDao.queryProvinceMove-Inline
### The error occurred while setting parameters
### SQL: SELECT m_date,province,rate         FROM tb_province_move         WHERE         m_date = ?
### Cause: java.sql.SQLSyntaxErrorException: Table 'covid19.tb_province_move' doesn't exist
; bad SQL grammar []; nested exception is java.sql.SQLSyntaxErrorException: Table 'covid19.tb_province_move' doesn't exist
	at com.sgg.covid19.dao.ProvinceMoveDaoTest.testQueryProvinceMoveList(ProvinceMoveDaoTest.java:22)
Caused by: java.sql.SQLSyntaxErrorException: Table 'covid19.tb_province_move' doesn't exist
	at com.sgg.covid19.dao.ProvinceMoveDaoTest.testQueryProvinceMoveList(ProvinceMoveDaoTest.java:22)

